kahuna/public/js/image/view.html

```diff
@@ -1,7 +1,7 @@
 <gr-top-bar>
     <gr-top-bar-nav>
-        <a class="top-bar-item side-padded clickable" ui:sref="search" title="Back to search">
-            <gr-icon-label gr-icon="search">Search</gr-icon-label>
+        <a class="top-bar-item side-padded clickable" ui:sref="search" gr:tooltip="Back to search" gr:tooltip-position="right">
```

Comment:
Is there really value in a tooltip that exactly repeats the label of the button? Or is that for the corner case of narrow browsers where the button was collapsed to the icon?